docs/docs/api/LayoutAnimations/AnimatedLayout.md

```diff
@@ -3,7 +3,7 @@ id: layout_animations_animated_layout
 title: <AnimatedLayout>
 sidebar_label: <AnimatedLayout>
 ---
-The component is responsible for observing changes in its subtree. It manages all its decendents' animations. Even if it has been removed. There is no need to nest one `AnimatedLayout` in another one. You can treat it as a `View` component.
+The component is responsible for observing changes in its subtree. It manages all its decendents' animations. Even if it has been removed. There is no need to nest one `AnimatedLayout` in another one because the higher one would already watch the subtree of the lower one. You can treat it as a `View` component.
```

Comment:
```suggestion
The component is responsible for observing changes in its subtree. There is no need to nest one `AnimatedLayout` in another one because the higher one would already watch the subtree of the lower one. You can treat it as a `View` component. `AnimatedLayout` manages all its descendants' animations while it's mounting and unmounting as well. 
```

I think directly mentioning unmounting/mounting is clearer than the previous one since it wasn't clear if it still manages them after removing from subtree.

Reply:
We may also have to clarify that in order to use AnimatedLayout people must use `Animated.View`.